### PR TITLE
[qtwebkit] Provide a way to turn off automatic content resizing when …

### PR DESCRIPTION
**Pull request by antseppa**
…geometry changes. Contributes to MER#1024

### CODE REVIEW

File: qtwebkit/Source/WebKit2/UIProcess/API/qt/qquickwebview.cpp [hunk 1]
```diff
@@ -1369,6 +1370,26 @@ void QQuickWebViewExperimental::setEnableInputFieldAnimation(bool enableInputFie
     emit enableInputFieldAnimationChanged();
 }
 
+bool QQuickWebViewExperimental::enableResizeContent() const
+{
+
```
Comment (rainemak) [inline qtwebkit/Source/WebKit2/UIProcess/API/qt/qquickwebview.cpp]: Why empty getter?
```
    Q_D(const QQuickWebView);
    return d->m_enableResizeContent;
```

Comment (antseppa) [inline qtwebkit/Source/WebKit2/UIProcess/API/qt/qquickwebview.cpp]: Oops, will update :)  (nothing really needed that on QML-side now)

**Comment by antseppa** [comment]
Updated: Fixed getter

**Comment by rainemak** [comment]
LGTM

**Pull request merged**